app/helpers/items_helper.rb

```diff
@@ -14,4 +14,8 @@ def document_presenter(document)
       # rubocop:enable Rails/HelperInstanceVariable
     end
   end
+
+  def license_options
```

Comment:
I think this coverage is screwed up because there is clearly a test for this: https://github.com/sul-dlss/argo/pull/2998/files#diff-03821fa1ffe288d0c703ce7b48be3553dd22405a7c777334e47f5330bbd1f2e3R13